src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -142,6 +142,14 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         }
     });
 
+    // Issue functions to the speechSynthesis interface after a delay; this
+    // makes the wrapper behave better when issuing commands in rapid succession
+    fluid.textToSpeech.asyncSpeechSynthesisControl = function (control, delay) {
```

Comment:
I agree with your assessment - closely spaced timeouts are notoriously unreliable. I was hoping this might be a sufficient patch for the moment, but a managed queue of pause/resume events won't take too long to implement.